psychopy/hardware/__init__.py

```diff
@@ -12,7 +12,7 @@
 except ImportError:
     from collections import Iterable
 
-__all__ = ['forp', 'cedrus', 'minolta', 'pr', 'crs', 'iolab', 'eyetracker']
+__all__ = ['forp', 'cedrus', 'minolta', 'pr', 'crs', 'iolab', 'eyetracker', 'mouse']
```

Comment:
keyboard too? Might be a small enough fix for this PR to include.

Reply:
Sure, good shout